sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/ui/EnterCodeActivity.kt

```diff
@@ -141,6 +142,10 @@ class EnterCodeActivity : DuckDuckGoActivity() {
                     override fun onPositiveButtonClicked() {
                         viewModel.onUserAcceptedJoiningNewAccount(it.encodedStringCode)
                     }
+
+                    override fun onDialogCancelled() {
```

Comment:
You likely need to add `onNegativeButtonClicked`; this is why the cancellation pixel wasn't sending when i was testing